x-pack/platform/plugins/shared/index_management/__jest__/client_integration/index_details_page/semantic_text_bannner.test.tsx

```diff
@@ -42,6 +42,10 @@ describe('When semantic_text is enabled', () => {
     expect(find('indexDetailsMappingsSemanticTextBanner').text()).toContain(
       'semantic_text field type now available!'
     );
+
+    expect(find('indexDetailsMappingsSemanticTextBanner').text()).toContain(
+      'Documents will be automatically chunked to fit model context limits.'
```

Comment:
```suggestion
      'Documents will be automatically chunked to fit model context limits, to avoid truncation.'
```

@saikatsarkar056 per discussion I'd make this tweak, with nice to have link if feasible:

 [Learn more](https://www.elastic.co/guide/en/elasticsearch/reference/current/inference-apis.html#infer-chunking-config)

Reply:
Updated